docker/scripts/v2/deploy-docker.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-sudo rm -rf docker/gethData/geth_data
```

Comment:
sudo might be needed if docker is added at sudo level

Reply:
added again `sudo`

Reply:
I don't think using sudo to run docker compose is a good approach